R/condition_message_linter.R

```diff
@@ -44,9 +44,8 @@
 condition_message_linter <- function() {
   translators <- c("packageStartupMessage", "message", "warning", "stop")
   xpath <- glue("
-  //SYMBOL_FUNCTION_CALL[
-    ({xp_text_in_table(translators)})
-    and not(preceding-sibling::OP-DOLLAR or preceding-sibling::OP-AT)
+  self::SYMBOL_FUNCTION_CALL[
+    not(preceding-sibling::OP-DOLLAR or preceding-sibling::OP-AT)
```

Comment:
WDYT about a new argument to `xml_find_function_calls()` to drop calls in extractions? I would do so by default. That would be another way to get at #2191 by only including extracted calls explicitly, since we usually don't want them.

Reply:
Sounds good for follow-up.
Need to think a bit about the best way to store additional metadata for function calls.
Usage in pipes seems interesting to track, too.

Reply:
#2428